mobile/src/main/java/ch/epfl/sdp/mobile/ui/profile/ProfileScreen.kt

```diff
@@ -66,14 +68,15 @@ fun <C : ChessMatch> ProfileScreen(
 @Composable
 fun <C : ChessMatch> ProfileHeader(
     state: VisitedProfileScreenState<C>,
-    modifier: Modifier = Modifier
+    modifier: Modifier = Modifier,
 ) {
 
   Column(
       modifier = modifier,
       horizontalAlignment = Alignment.CenterHorizontally,
       verticalArrangement = Arrangement.spacedBy(16.dp),
   ) {
+    BackButton(state::onBack, modifier.align(Alignment.Start))

```

Comment:
Bug : using `modifier` instead of `Modifier`.

```suggestion
    BackButton(state::onBack, Modifier.align(Alignment.Start))

```

Reply:
Note : you should change line 45 of this file to be :

```diff
- header = { ProfileHeader(state, Modifier.padding(vertical = 16.dp)) },
+ header = { ProfileHeader(state, Modifier.padding(16.dp).fillMaxWidth()) },
```

so the right paddings are applied.

Reply:
👍